python/vespa/vespa/package.py

```diff
@@ -504,122 +513,208 @@ def __init__(self, tenant: str, application: str, key_location: str) -> None:
+            if update["active"]:
                 sleep(1)
             else:
-                status = update['status']
-                if   status == 'success':                    return
-                elif status == 'error':                      raise RuntimeError("Unexpected error during deployment; see log for details")
-                elif status == 'aborted':                    raise RuntimeError("Deployment was aborted, probably by a newer deployment")
-                elif status == 'outOfCapacity':              raise RuntimeError("No capacity left in zone; please contact the Vespa team")
-                elif status == 'deploymentFailed':           raise RuntimeError("Deployment failed; see log for details")
-                elif status == 'installationFailed':         raise RuntimeError("Installation failed; see Vespa log for details")
-                elif status == 'running':                    raise RuntimeError("Deployment not completed")
-                elif status == 'endpointCertificateTimeout': raise RuntimeError("Endpoint certificate not ready in time; please contact Vespa team")
-                elif status == 'testFailure':                raise RuntimeError("Unexpected status; tests are not run for manual deployments")
-                else:                                        raise RuntimeError("Unexpected status '" + status + "'")
+                status = update["status"]
+                if status == "success":
+                    return
+                elif status == "error":
+                    raise RuntimeError(
+                        "Unexpected error during deployment; see log for details"
+                    )
+                elif status == "aborted":
+                    raise RuntimeError(
+                        "Deployment was aborted, probably by a newer deployment"
+                    )
+                elif status == "outOfCapacity":
+                    raise RuntimeError(
+                        "No capacity left in zone; please contact the Vespa team"
+                    )
+                elif status == "deploymentFailed":
+                    raise RuntimeError("Deployment failed; see log for details")
+                elif status == "installationFailed":
+                    raise RuntimeError("Installation failed; see Vespa log for details")
+                elif status == "running":
+                    raise RuntimeError("Deployment not completed")
+                elif status == "endpointCertificateTimeout":
+                    raise RuntimeError(
+                        "Endpoint certificate not ready in time; please contact Vespa team"
+                    )
+                elif status == "testFailure":
+                    raise RuntimeError(
+                        "Unexpected status; tests are not run for manual deployments"
+                    )
+                else:
+                    raise RuntimeError("Unexpected status '" + status + "'")
```

Comment:
Hehe, this is why I hate autoformatters :)

Reply:
@oyving sold me on `black` a while ago, now it is hard to live without it.

Reply:
Yeah, pros and cons :)